ivy/functional/frontends/torch/pointwise_ops.py

```diff
@@ -181,7 +181,8 @@ def div(input, other, *, rounding_mode=None, out=None):
         return ivy.divide(input, other, out=out)
 
 
-@with_unsupported_dtypes({"2.1.2 and below": ("float16", "complex")}, "torch")
+@with_unsupported_dtypes({"2.1.0 and below": ("float16", "complex")}, "torch")
+@with_unsupported_dtypes({"2.5.1 and below": ("bfloat16",)}, "paddle")
```

Comment:
Hey @yopknopixx, actually the use of the `with_unsupported_dtypes` decorator in the frontends primarily refers to the datatypes that the function in the actual framework that the frontend belongs to doesn't support. For specifying backend-specific unsupported dtypes, we should update the corresponding decorator in the backend implementation (e.g. the decorator in the `paddle` backend implementation of `ivy.erf`), same applies to `ivy.log`. Thanks 😄